### PR TITLE
SAK-29707 -Displays confirmation message when Joinable Group is deleted.

### PR DESCRIPTION
**Pull request by ouit0408**
Confirm message comes when user clicks "Delete all groups" button and for
that have created 'JoinableGroupDelProducer.java',
'JoinableGroupDelViewParameters.java' and 'JoinableGroupDel.html'.Also, have
edited 'CreateJoinableGroups.html' and 'CreateJoinableGroupsProducer.java'.

### CODE REVIEW

**Comment by buckett** [comment]
This indentation on this is wrong and needs cleaning up.

**Comment by buckett** [comment]
The indentation on this still needs cleaning up.

**Comment by buckett** [comment]
This has merge conflicts.

File: site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/JoinableGroupDelViewParameters.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import uk.org.ponder.rsf.viewstate.SimpleViewParameters;
+
+public class JoinableGroupDelViewParameters extends SimpleViewParameters{
+	public String id;
+
+    public JoinableGroupDelViewParameters(String id) {
+		super();
+		this.id = id;
+	}
```
Comment (bjones86) [inline site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/JoinableGroupDelViewParameters.java]: Looks like a strange mix of tabs/spaces in this file.

**Comment by ouit0408** [comment]
as this has been fixed in sakai 11 , closing the pull request.

**Comment by buckett** [comment]
I don't think Sakai 11 does have this functionality, there is a confirmation when deleting the group, but there isn't in Sakai a confirmation when removing the "joinable set".

**Pull request closed without merging**